packages/module/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.js
+*.d.ts
```

Comment:
why type definition files should be ignored here?

Reply:
All. During a build step and publishing step i compile ts to js/d.ts files. It's just to prevent autocommiting them.